measurecrack.py

```diff
@@ -0,0 +1,175 @@
+import cv2, imutils
+import numpy as np
+
+# Measures crack using raio method
+def measure_crack(image):
+    #Get dimensions of blue rectangle
+    width, height = blue_rectangle(image)
+    
+    # Return larger dimension. Multiply ratio by 1.85
+    if height > width:
+        return (height/width)*1.9
+    else:
+        return (width/height)*1.9
+
+# Finds dimensions of a blue rectangle    	
+def blue_rectangle(image):
+    # Convert BGR to HSV
+    hsv = cv2.cvtColor(image, cv2.COLOR_BGR2HSV)
+
+    # define range of blue color in HSV
```

Comment:
these numbers are probably fine since they are defining colors and youre basically putting them into constants.

nevertheless, i would change the `lower_blue` and `upper_blue` to all caps since they are constant